### PR TITLE
capifony and html fixes

### PR DESCRIPTION
**Pull request by liuggio**

### CODE REVIEW

File: app/config/deploy.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+set :branch, "master"
+set :git_shallow_clone, 1
+set :model_manager, "doctrine"
+set  :keep_releases,  3
+set  :use_sudo,       false
+set :use_composer, true
+
+
+set :repository,  "git@github.com:PUGX/botrelli.git"
+set :scm,         :git
```
Comment (bigfoot90) [inline app/config/deploy.rb:24]: Double variable assignment

File: app/config/deploy.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+# Or: `accurev`, `bzr`, `cvs`, `darcs`, `subversion`, `mercurial`, `perforce`, or `none`
+
+set :model_manager, "doctrine"
```
Comment (bigfoot90) [inline app/config/deploy.rb:27]: Double variable assignment

File: src/Bot.php [hunk 3]
```diff
@@ -43,7 +43,7 @@ public function execute(Package $package, $dryRun = false)
 
         $step      = new Step\CloneLocally($gitWrapper, $this->dispatcher);
         $step->execute($localPackage);
-
+x
```
Comment (bigfoot90) [inline src/Bot.php:46]: What is this ?

Comment (liuggio) [inline src/Bot.php:46]: a bug :D
this was fixed thanks

File: app/config/deploy.rb (hunk 1 above)
Comment (liuggio) [inline app/config/deploy.rb:24]: great :)
could you provide  a PR for those 2 double ass.?

Comment (bigfoot90) [inline app/config/deploy.rb:24]: I'm on holiday now, if you can wait two weeks i can made a PR.

**Pull request merged**